### ISSUE TITLE
Is that possible to concat multiple HTML to make one PDF ?

### ISSUE THREAD

**Issue by fouzko** [marcbachmann/node-html-pdf#104]
Or concat some PDFs in juste one ?!

**Comment by marcbachmann** [comment]
no. please use another tool for that.

**Comment by scurker** [comment]
@fouzko Yes, it's possible to print different pages in one pdf with css - depending on how your html is setup.

With a structure like this:

```html
<style>
  .page { page-break-before: always }
</style>
<div class="page">
...
</div>
<div class="page">
...
</div>
```

...anything contained within those page divs would be different pages in the PDF. I know this isn't exactly what you're asking, but the closest way to simulate "multiple" pdfs.

**Comment by bigamasta** [comment]
@scurker
Helped a lot.
Cheers.

**Comment by marcbachmann** [comment]
We really should put together some best practices. Abd write down all the limitations. Phantomjs and also chromium have many limitations which aren’t that obvious.

**Comment by SamMatthewsIsACommonName** [comment]
Mine are still seeming to ignore these break rules. Would the fact I'm using A4 size make a difference to how this would behave?

**Comment by NikhilMutkekar** [comment]
Thanks It did help!

**Comment by karamjb** [comment]
Thanks @scurker 
`<style>
  .page { page-break-before: always }
</style>` helped my a lot
